superx/routes/home.py

```diff
@@ -1,7 +1,7 @@
 """
 import from flask and models and db
 """
-# pylint: disable=redefined-outer-name, no-member, consider-using-enumerate
+# pylint: disable=redefined-outer-name, no-member, consider-using-enumerate, line-too-long
```

Comment:
You can't turn off every check that you don't like.
Why not fix the line so it is less than 80 chars. (by using \ to break the line into multiple lines.)

Reply:
your'e correct, I fixed the problem and removed the disabling